.coafile

```diff
@@ -29,6 +29,7 @@ bears = SpaceConsistencyBear
 
 [commit]
 bears = GitCommitBear
+issue_checks = True
```

Comment:
1. I think this should go into a separate commit
2. you don't define `issue_checks` as a settings, instead you have
    ```
    detect_host: bool=False,
    require_url: str=None,
    keyword_regexp: str=None
    ```
    So either unify them to an `issue_checks` setting, otherwise you need to use those settings^^